sql/core/src/test/scala/org/apache/spark/sql/MetadataCacheSuite.scala

```diff
@@ -55,8 +56,8 @@ abstract class MetadataCacheSuite extends QueryTest with SharedSparkSession {
       val e = intercept[SparkException] {
         df.count()
       }
-      assert(e.getMessage.contains("FileNotFoundException"))
-      assert(e.getMessage.contains("recreating the Dataset/DataFrame involved"))
+      assertExceptionMessage(e, "FileNotFoundException")
```

Comment:
`assertErrorMsgWithAQE`?

Reply:
The helper method can also work for normal cases without AQE but adding `AQE` just sounds like it's only used by AQE?